tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py

```diff
@@ -78,19 +78,21 @@ def random_group_id():
     return f"test-consumer-{random.randint(0, 2 ** 16)}"
 
 
-@pytest.mark.skip(reason="flaky: INGEST-1644")
 @pytest.mark.django_db(transaction=True)
+@pytest.mark.parametrize("run", range(10))  # FIXME: Remove before merging
 @pytest.mark.parametrize(
     "executor",
     [pytest.param(None, id="synchronous"), pytest.param(ThreadPoolExecutor(), id="asynchronous")],
 )
 def test_ingest_consumer_reads_from_topic_and_calls_celery_task(
+    run,
     executor,
     task_runner,
     kafka_producer,
     kafka_admin,
     default_project,
     get_test_message,
+    random_group_id,
```

Comment:
By declaring this function argument, pytest actually evaluates the fixture, so when we pass it on line 113, it contains  a generated random string instead of a function.